transform/schemas/properties/relations.py

```diff
@@ -0,0 +1,63 @@
+import os
+from schemas.properties.data import ID, LEGALNAME
+from schemas.properties.env import *
+from utils.loader import load_env_vars
+
+env_vars = load_env_vars()
+
+# HELPER COLUM NAMES
```

Comment:
Those are all data-related properties. I would consider moving all data-related properties to data.py under a comment # relations- or something in that manner

Reply:
Done